### DISCUSSION TITLE
Missing png / icons in apk

### DISCUSSION THREAD

**Discussion by adrianTNT** [iBotPeaches/Apktool#3442] [General]
Does anyone know why some newish apks do not return the actual APK png icons / images ?
I see some images, but parts of layout, no icon.
For example: [https://apps.evozi.com/apk-downloader/?id=com.android.chrome](https://apps.evozi.com/apk-downloader/?id=com.android.chrome)

**Comment by iBotPeaches** [comment]
Converted to discussion as not really a template followed bug report. Don't have a lot of details to go on here, but split apks and the movement to adaptive/vectors would be less obvious of a png/jpg, but still there.
